one_collect/src/helpers/exporting/scripting.rs

```diff
@@ -905,7 +905,7 @@ impl ScriptedUniversalExporter {
         self.rhai_engine().register_fn(
             "record_event",
             move |event: ScriptEvent| -> Result<(), Box<EvalAltResult>> {
-                info!("Script called record_event");
+                info!("Recording event: ");
```

Comment:
Please include the event name here.  You likely need to move this lower so that you have access to the event.